src/contrib/persistence/Akka.Persistence.Sql.Common/Snapshot/QueryExecutor.cs

```diff
@@ -147,6 +154,7 @@ public class QueryConfiguration
             SerializerIdColumnName = serializerIdColumnName;
             Timeout = timeout;
             DefaultSerializer = defaultSerializer;
+            UseSequentialAccess = useSequentialAccess;
```

Comment:
What's the default value of this? Is it left up to the plugin implementation? Should we add it to some of the built-in HOCON files?

Reply:
Sorry for the late response.
Default value is false = current behavior
As the configuration is read by the plugin implementations it is basically up to the plugins. Therefore I'm not sure if it makes sense to put into some HOCO file on Akka level.
So I'm not sure if further changes are needed.